retrofit/src/main/java/retrofit2/RequestFactory.java

```diff
@@ -712,6 +713,24 @@ private Headers parseHeaders(String[] headers) {
         }
         gotBody = true;
         return new ParameterHandler.Body<>(method, p, converter);
+
```

Comment:
nit: unnecessary new line

Reply:
It's intentional to separate the branches.